doc/hotspot-style.md

```diff
@@ -138,6 +138,11 @@ a .inline.hpp file.
 * .inline.hpp files should only be included in .cpp or .inline.hpp
 files.
 
+* All .inline.hpp files should include their corresponding .hpp file
+as the first include line. Externally used declarations should be put
```

Comment:
Do we need to spell out it should be first unless precompiled.hpp is included?

Reply:
We should never include precompiled.hpp from an .inline.hpp file, so I don't think we need to state that.

Reply:
Ah. Right. :)